internal/sync/service.go

```
@@ -0,0 +1,429 @@
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"text/template"
+)
+
+// ServiceManager represents a process management system.
+type ServiceManager string
+
+const (
+	ServiceManagerSystemd ServiceManager = "systemd"
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gofumpt)